lua/lspinstall/servers/java.lua

```diff
@@ -0,0 +1,80 @@
+local config = require'lspconfig'.jdtls.document_config
+require'lspconfig/configs'.jdtls = nil -- important, immediately unset the loaded config again
+
+config.default_config.cmd[1] = {
```

Comment:
This should be `cmd`, are you *sure* `cmd[1]` worked here? If so, something is off.